cli/cmd/start.go

```diff
@@ -0,0 +1,154 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"lido-aragon/pkg/aragon"
+	"lido-aragon/pkg/daemon"
+	"lido-aragon/pkg/deploy"
+	"log"
+	"os"
+
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+	"github.com/ttacon/chalk"
+)
+
+func init() {
+	rootCmd.AddCommand(startCmd)
+	rootCmd.AddCommand(startAragon)
+	rootCmd.AddCommand(startNode)
+
+	startCmd.AddCommand(startAllCmd)
+	startCmd.AddCommand(startForkCmd)
+}
+
+var startAragon = &cobra.Command{
+	Use:   "aragon",
+	Short: "Start aragon client only",
+	Run: func(cmd *cobra.Command, args []string) {
+		os.Setenv("NETWORK_NAME", Lido.NetworkName)
+
+		var aragonCmd aragon.AragonNetwork
+
+		if Lido.NetworkName == "mainnet" {
+			aragonCmd = aragon.CMD_MAINNET
+		}
+
+		pterm.Info.Printf("Network: %s\n", Lido.NetworkName)
+
+		Lido.Deploy.DeployedFile, _ = getDeployedFile(Lido.NetworkName)
+
+		err := Lido.AragonClient.Start(aragonCmd, Lido.LidoApps.AppsLocator, Lido.Deploy.DeployedFile)
+		if err != nil {
+			return
+		}
+
+		pterm.Println()
+		pterm.FgWhite.Println("ARAGON_ENS_REGISTRY_ADDRESS = " + Lido.AragonClient.EnsRegistry)
+		pterm.FgWhite.Println("Start aragon at: ", chalk.Yellow, Lido.AragonClient.RunningUrl+"/#/"+Lido.Deploy.DeployedFile.DaoAddress)
+
+		daemon.WaitCtrlC()
+	},
+}
+
+var startCmd = &cobra.Command{
+	Use:   "start",
+	Short: "Start local or form env",
+}
+
+var startAllCmd = &cobra.Command{
+	Use:   "all",
+	Short: "Start hardhat node, start ipfs, deploy contracts, start lido apps, start aragon",
+	Run: func(cmd *cobra.Command, args []string) {
+
+		os.Setenv("NETWORK_NAME", Lido.NetworkName)
+
+		Lido.Contracts.Start()
+		Lido.HardhatNode.Start("")
```

Comment:
I got the following error when trying to restart already terminated lido-aragon:

>  `lido-aragon start all -v`
...
All done!
 SUCCESS  Contracs: compile...done                                                                                                                                                                                                     
  ERROR   Hardhat node: Starting...                                                                                                                                                                                                    
  ERROR   Error: listen EADDRINUSE: address already in use 127.0.0.1:8545
              at Server.setupListenHandle [as _listen2] (net.js:1316:16)
              at listenInCluster (net.js:1364:12)
              at doListen (net.js:1501:7)
              at processTicksAndRejections (internal/process/task_queues.js:85:21)
▀  IPFS: Starting... (1s)Initializing daemon...

Reply:
It's reproducible after abnormal termination (via ctrl+c in the middle of something or another thrown error). It's just a `hardhat node` that sits on the port and could be easily killed by the SIGTERM signal.
Also, sometimes we need to delete the `deployed-localhost.json` to recover completely.